.github/workflows/destroy-expired-environment.yml

```diff
@@ -0,0 +1,59 @@
+name: Destroy Expired Environment
+
+on:
+  # TODO: check if workflow_dispatch is needed
+  workflow_dispatch:
+  # TODO: remove pull_request after debugging
+  pull_request:
+    branches:
+      - main
+      - "[0-9]+.[0-9]+"
+    types: [opened, synchronize, reopened]
+
```

Comment:
I'm using this option to run the workflow temporarily. After the review and necessary fixes, it will be removed.